### PR TITLE
Apply zentyalcloud rename also in doc

### PR DESCRIPTION
**Pull request by exekias**

### CODE REVIEW

**Comment by mateobur** [comment]
:+1:

File: doc/provision.rst [hunk 1]
```diff
@@ -40,8 +40,10 @@ To create an admin user for the parent company execute:
 
 .. code-block:: console
 
-  root@samba:~# zentyalcloud create companyadmin 415b55ce6c114132b01bd1edd80f37f7
+  root@samba:~# zinc create user --admin 415b55ce6c114132b01bd1edd80f37f7
   Username: admin
+  Frist name: John
```
Comment (mateobur) [inline doc/provision.rst]: s/Frist/First

File: doc/provision.rst [hunk 2]
```diff
@@ -29,7 +29,7 @@ You can create any number of child companies using this ID:
 
 .. code-block:: console
 
-  root@samba:~# zentyalcloud create company --mail --partner 415b55ce6c114132b01bd1edd80f37f7
+  root@samba:~# zinc create company --partner 415b55ce6c114132b01bd1edd80f37f7
```
Comment (figarocorso) [inline doc/provision.rst]: It may be not necessary to create this company using the command line.

Besides this comment, :+1: .

Comment (exekias) [inline doc/provision.rst]: Yep, it's an example so they can mentaly connect GUI with commandline

**Pull request merged**